loglist2/loglist2.go

```diff
@@ -136,6 +136,59 @@ type TreeHead struct {
 	TreeSize int64 `json:"tree_size"`
 }
 
+// String method returns printable name of the state.
+func (ls *LogStates) String() string {
+	names := [...]string{
+		"Empty",
+		"Pending",
+		"Qualified",
+		"Usable",
+		"Frozen",
+		"Retired",
+		"Rejected"}
+	switch {
+	case ls == nil:
+		return names[0]
+	case ls.Pending != nil:
+		return names[1]
+	case ls.Qualified != nil:
+		return names[2]
+	case ls.Usable != nil:
+		return names[3]
+	case ls.Frozen != nil:
+		return names[4]
+	case ls.Retired != nil:
+		return names[5]
+	case ls.Rejected != nil:
+		return names[6]
+	default:
+		return names[0]
+	}
+}
+
+// Active picks the set-up state. If multiple states are set (not expected) picks one of them.
+func (ls *LogStates) Active() (*LogState, *FrozenLogState) {
```

Comment:
Do you have an example use case for this method? I'm struggling to see how you'd use it.

Reply:
"in the next series..." That's why we have 2 methods. Probably, some enum-returning func is more obvious than stringer.

Reply:
Are you saying you're planning on using the `String()` method to determine which state a log is in? That seems less than ideal, since a change to the strings either here or in your code would break the comparisons.